client/components/Querries.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+// import Collapsible from 'react-collapsible';
+// import { Projects, Project, ProjectQuery } from './Interfaces';
+import { ProjectQuery } from './Interfaces';
+
+export interface IProps {
+    // projects: Projects['projects'];
+    // test: any;
+    arrowTime: string | undefined;
+    arrowDepth: string | undefined;
+    arrowComplexity: string | undefined;
+    setToggle: any;
+    time: boolean;
+    depth: boolean;
+    complexity: boolean;
+    sortByTime: any;
+    sortByDepth: any;
+    sortByComplexity: any;
+    queries: ProjectQuery[] | undefined;
+}
+// eslint-disable-next-line react/function-component-definition
+const Querries: React.FC<IProps> = ({
+    arrowTime,
+    arrowDepth,
+    arrowComplexity,
+    setToggle,
+    time,
+    depth,
+    complexity,
+    sortByTime,
+    sortByDepth,
+    sortByComplexity,
+    queries,
+}) => (
+    // const Querries: React.FC<IProps> = ({ projects, test, sortByNameAsc, queries }) => (
+    <div>
+        <div>
+            <div id="loggerBtnWrapper">
+                <div
```

Comment:
If possible, let's pin the header to the top of the pane. This way a developer doesn't need to scroll to the top to sort the queries by another value. In this case, it may be worth having the list jump to the top after it is sorted.

Reply:
Will need more clarification on this comment..